mbox-operator/molecule/default/asserts/koji-hub.yml

```
@@ -0,0 +1,55 @@
+---
+
+- name: Verify
+  hosts: localhost
+  connection: local
+  vars:
+    ansible_python_interpreter: '{{ ansible_playbook_python }}'
+    koji_configmap: koji-hub-configmap
+    kojihub_cacert_secret: cacert
+    kojihub_pvc: koji-hub-pvc
+  tasks:
+      - block:
+        - name: 'TEST: koji_hub.configmap'
+          k8s_info:
+            api_version: v1
+            kind: ConfigMap
+            namespace: "{{ namespace }}"
+            name: "{{ koji_configmap }}"
+          register: koji_configmap
+        - assert:
+            that:
+              - koji_configmap.resources|length == 1
+              - koji_configmap.resources[0].metadata.labels['app'] == 'koji-hub'
+              - "'httpd.conf' in koji_configmap.resources[0].data"
+              - "'hub.conf' in koji_configmap.resources[0].data"
+              - "'kojiweb.conf' in koji_configmap.resources[0].data"
+      - block:
+          - name: 'TEST: kojihub.secret.cacert'
+            k8s_info:
+              api_version: v1
+              kind: Secret
+              namespace: "{{ namespace }}"
+              name: "{{ kojihub_cacert_secret }}"
+            register: kojihub_secrets
+          - assert:
+              that:
+                - kojihub_secrets.resources|length == 1                
+                - kojihub_secrets.resources[0].metadata.labels['app'] == 'koji-hub'
+                - "'ca.crt' in kojihub_secrets.resources[0].data"
```

```suggestion
                - "'cert' in kojihub_secrets.resources[0].data"
```
In `koji-hub` role main this is fixed `cert` value.